src/library/scala/collection/mutable/ArrayOps.scala

```diff
@@ -296,6 +335,12 @@ final class ofUnit(override val repr: Array[Unit]) extends AnyVal with ArrayOps[
     override protected[this] def thisCollection: WrappedArray[Unit] = new WrappedArray.ofUnit(repr)
     override protected[this] def toCollection(repr: Array[Unit]): WrappedArray[Unit] = new WrappedArray.ofUnit(repr)
     override protected[this] def newBuilder = new ArrayBuilder.ofUnit
+    protected override def emptyImpl = emptyUnitArray
+    protected override def sliceImpl(from: Int, until: Int) = {
+      // cant use util.Arrays.copyOfRange[Unit](repr, from, until) - Unit is special and doesnt compile
+      // but Unit and null are analogous so values are not important to copy across
+      Array[Unit](until-from)
```

Comment:
If you wanted to actually copy the array (to preserve whether the contents are `null` or instances of `scala.runtime.BoxedUnit`--perversely, either one is okay--you could
```
java.util.Arrays.copyOfRange(
  repr.asInstanceOf[Array[scala.runtime.BoxedUnit]],
  from,
  until
).asInstanceOf[Array[Unit]]
```
(maybe on a single line--I'm just spreading it out here to make it easier to see).